Specification/4._CAN_bus_transport_layer.md

```diff
@@ -19,43 +19,39 @@ UAVCAN defines the following types of transfers:
 Both message and service transfers can be further distinguished between:
 
 * Single-frame transfer - a transfer that is entirely contained in a single CAN frame.
-* Multi-frame transfer - a transfer whose payload is distributed over multiple CAN frames.
+* Multi-frame transfer - a transfer that has its payload distributed over multiple CAN frames.
 
 The following properties are common to all types of transfer:
 
 Property        | Description
 ----------------|------------------------------------------------------------------------------------------------------
 Payload         | The serialized data structure
-Data type ID    | This identifier indicates how the data structure should be interpreted
-Priority        | See below
-Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from others
+Data type ID    | An identifier that indicates how the data structure should be interpreted
+Priority        | An positive integer value that defines the message urgency (higher numbers indicate reduced priority). Higher priority transfers can delay transmission of loser priority transfers. 
```

Comment:
Is 0 the highest priority? If so, then we might simply say that rather than "higher numbers indicate reduced priority"

Reply:
That can be done, indeed. Also please note:

> An positive integer

Shouldn't it be "A positive integer"?

> delay transmission of loser priority transfers

Lower.

Reply:
Thanks. Yes - sorry for injecting bugs into the docs :-(